openeo/internal/filter_bbox.json

```diff
@@ -0,0 +1,123 @@
+{
```

Comment:
I think we should avoid mirroring json files from openeo-api and -processes

Reply:
Sry that was just for testing, so should have never bin in the repository:
Removed in the last commit